### DISCUSSION TITLE
请问在X6当前版本如何实现沿边动画

### DISCUSSION THREAD

**Discussion by zzyyu** [antvis/X6#3358] [Q&A] [answered]
在X6 [V1版本文档](https://x6.antv.vision/zh/docs/tutorial/advanced/animation#%E6%B2%BF%E8%BE%B9%E8%BF%90%E5%8A%A8%E7%9A%84%E5%8A%A8%E7%94%BB)中提供了沿边动画的内容
在V2版本中无法使用，在issues中发现的[demo](https://codesandbox.io/s/epic-hypatia-70blmk?file=/src/app.tsx)会报错`view.sendToken is not a function`
查看源码也未找到sendToken相关内容
请问是在V2版本取消掉沿边动画了吗？
如果未取消，请问如何实现？

**Comment by NewByVector** [comment, marked as answer]
X6 2.0 将会将动画封装成插件，现在还没有完成。在 2.x 中，可以将 1.x sendToken 方法拷贝到本地自己实现。

**Reply by AllenMRbai** [reply]
原来如此。我还以为2.x不支持动画了呢。
